### ISSUE TITLE
Can you provide training code?

### ISSUE THREAD

**Issue by fightlee01** [brijeshiitg/Pytorch-implementation-of-SRNet#8]
Thank you very much for your open source pytorch version of SRNET network code. Can you provide training code? I use your code for training and find that the network does not converge all the time.

**Comment by davidggz** [comment]
I'm having the same issue; I managed to get two models with 80% accuracy in validation but the network struggle a lot to get to losses lower than 0.69. I'm using Keras with a sigmoid activation function and binary crossentropy. 

@fightlee01 did you manage to get the network to converge all the time?

**Comment by brijeshiitg** [comment]
training code has been uploaded
